cms/topmenu.json

```diff
@@ -1,8 +1,7 @@
 {
   "items": [
-    {"Home": "http://apps.loklak.org"},
+  	{"About": "http://loklak.net/about"},
```

Comment:
Can you remove the additional whitespaces or tabs added so as to provide uniformity pls.